services/core/shared/core/compat/sdk_v5/paramUtils.js

```diff
@@ -19,7 +19,12 @@ const {
 
 const normalizeRangeParam = (params, property) => {
 	if (typeof params[property] === 'string' && params[property].includes(':')) {
-		const [from, to] = params[property].split(':');
+		const [fromStr, toStr] = params[property].split(':');
+
+		const from = Number(fromStr);
+		const to = Number(toStr);
+
+		if (Number.isNaN(from) || Number.isNaN(to)) throw new ValidationException(`Invalid (non-numeric) '${property}' range values supplied: ${params[property]}.`);
 		if (from && to && from > to) throw new ValidationException(`From ${property} cannot be greater than to ${property}.`);
 
 		if (!params.propBetweens) params.propBetweens = [];
```

Comment:
To support partial ranges (`100:` or `:100`) appropriately, please consider replacing the next line with the following. With the current changes, the unsupplied `to` or `from` defaults to `0`. Also, we'll need to add a couple of test cases corresponding to the suggested changes.

```
		params.propBetweens.push({ property, from, to });
```
to
```
		if (fromStr === '') {
			params.propBetweens.push({ property, to });
		} else if (toStr === '') {
			params.propBetweens.push({ property, from });
		} else {
			params.propBetweens.push({ property, from, to });
		}
```

Reply:
@sameersubudhi Can you please provide the issue number?

Reply:
Created the following, with the exact information: https://github.com/LiskHQ/lisk-service/issues/1056